src/main/java/com/minecolonies/coremod/commands/ActionMenuType.java

```diff
@@ -0,0 +1,300 @@
+            ForgePermissionNodes.DisableBarbarianSpawns,
+            DisableBarbarianSpawnsCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required),
+            new ActionArgument("disableSpawns", ActionArgumentType.BOOLEAN, ActionArgumentType.Is.Required)
+            )),
+    AddOfficer(new ActionMenu(
+            "AddOfficer",
+            "addOfficer",
+            ForgePermissionNodes.AddOfficer,
+            AddOfficerCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required),
+            new ActionArgument("player", ActionArgumentType.Player, ActionArgumentType.Is.Required)
+            )),
+    RefreshColony(new ActionMenu(
+            "RefreshColony",
+            "refresh",
+            ForgePermissionNodes.RefreshColony,
+            RefreshColonyCommand.class,
+            // TODO: need OR() ActionArgumentType
+            new ActionArgument("player", ActionArgumentType.Player, ActionArgumentType.Is.Optional),
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Optional)
+            )),
+    ChangeColonyOwner(new ActionMenu(
+            "Ownership Change",
+            "ownerchange",
+            ForgePermissionNodes.ChangeColonyOwner,
+            ChangeColonyOwnerCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required),
+            new ActionArgument("player", ActionArgumentType.Player, ActionArgumentType.Is.Required)
+            )),
+    ColonyTeleport(new ActionMenu(
+            "ColonyTeleport",
+            "teleport",
+            ForgePermissionNodes.ColonyTeleport,
+            ColonyTeleportCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required)
+            )),
+    MakeNotAutoDeletable(new ActionMenu(
+            "MakeNotAutoDeletable",
+            "deletable",
+            ForgePermissionNodes.MakeNotAutoDeletable,
+            com.minecolonies.coremod.commands.colonycommands.MakeNotAutoDeletable.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required),
+            new ActionArgument("canBeDeleted", ActionArgumentType.BOOLEAN, ActionArgumentType.Is.Required)
+            )),
+    DoRaidNow(new ActionMenu(
+            "DoRaidNow",
+            "raid",
+            ForgePermissionNodes.DoRaidNow,
+            DoRaidNowCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required)
+            )),
+    DoRaidTonight(new ActionMenu(
+            "DoRaidTonight",
+            "raid-tonight",
+            ForgePermissionNodes.DoRaidTonight,
+            DoRaidTonightCommand.class,
+            new ActionArgument("colony", ActionArgumentType.Colony, ActionArgumentType.Is.Required)
+            )),
+    RSReset(new ActionMenu(
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00115)